lib/ddtrace/contrib/mongodb/subscribers.rb

```diff
@@ -0,0 +1,104 @@
+module Datadog
+  module Contrib
+    # MongoDB module includes classes and functions to instrument MongoDB clients
+    module MongoDB
+      # `MongoCommandSubscriber` listens to all events from the `Monitoring`
+      # system available in the Mongo driver.
+      class MongoCommandSubscriber
+        # rubocop:disable Metrics/MethodLength
+        # rubocop:disable Metrics/AbcSize
+        # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
don't like these ignores, but the class doesn't seem really complex; really opened to improvements here

Reply:
👍